### ISSUE TITLE
Rename tiddler hackability

### ISSUE THREAD

**Issue by AnthonyMuscio** [Jermolene/TiddlyWiki5#3649]
For a future version of TiddlyWiki,( not specifically the 5.1.19 patch release), could we have an additional system tag that transcludes tiddlers so tagged in the case of tiddler renames. It could have being used to introduce and hide "Show backlinks on rename (#3534) " but will allow the hack-ability of the rename function. 

Users could develop their own method to respond to tiddler renames, which are effectively "keys" to information contained in a tiddler. It could be used to provide suggested titles, or enforce referential integrity along with introducing actual methods to resolve references on rename.

Regards
Tony

**Comment by flibbles** [comment]
Hello!

Does the new [Relink plugin](https://flibbles.github.io/tw5-relink/), which automatically relinks all tiddlers on a rename, provide a solution to your problem?

**Comment by AnthonyMuscio** [comment]
Yes it does, but it does not open up the hackability. For example I may want to add a logging method to capture all renames, or update an index or something.

Relink was one of the key things I wanted to achieve and you have done that thanks, but it does not change the need for the hackability.

If I am not wrong the relink plugin needs to overwrite one of more system tiddlers.  It would be better if it too could use a hackability method rather than overwites core tiddlers.

**Comment by flibbles** [comment]
Ah. I _think_ I understand what you're after, and no, Relink isn't the solution for that, just something that would benefit from such a solution. I did have to overwrite some already-defined wiki methods for this.

**Comment by AnthonyMuscio** [comment]
If this change proceeds you may be able to avoid overwriting system tiddlers in a future version.
